### ISSUE TITLE
"Update Site Fix" did not fix the site.

### ISSUE THREAD

**Issue by PancakeTAS** [PyvesB/eclipse-planet-themes#9]
Eclipse 2021-12 throws the following error at me:

'Installing Software' has encountered a problem.
```text
An error occurred while collecting items to be installed
session context was:(profile=C__Users_games_Documents_EclipseIDE_Eclipse_eclipse, phase=org.eclipse.equinox.internal.p2.engine.phases.Collect, operand=, action=).
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.swt_0.6.0.20200802-1642.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.swt_0.6.0.20200802-1642.jar
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.swt.win_0.6.0.20200802-1642.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.swt.win_0.6.0.20200802-1642.jar
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.ui.swt.theme_0.8.0.20200802-1642.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.ui.swt.theme_0.8.0.20200802-1642.jar
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.ui.swt.theme.win_0.8.0.20200802-1642.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.eclipse.ui.swt.theme.win_0.8.0.20200802-1642.jar
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.util_0.1.0.20200802-1642.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/com.codeaffine.util_0.1.0.20200802-1642.jar
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/io.github.pyvesb.eclipse_planet_themes_0.2.2.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/plugins/io.github.pyvesb.eclipse_planet_themes_0.2.2.jar
Artifact not found: https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/features/io.github.pyvesb.eclipse_planet_themes_0.2.2.202110171330.jar.
https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/features/io.github.pyvesb.eclipse_planet_themes_0.2.2.202110171330.jar
```

The new [page](https://cdn.jsdelivr.net/gh/PyvesB/eclipse-planet-themes@updatesite/features/io.github.pyvesb.eclipse_planet_themes_0.2.2.202110171330.jar) throws "Forbidden" at the user

**Comment by PyvesB** [comment]
Hello @MCPfannkuchenYT ! 👋🏻 

The installation through Eclipse works fine on my machine, but as it's also the computer I use to develop the plugin itself, maybe all the files are saved in local .p2 caches... Could you please try the following URL and tell me whether it works any better?

https://gitcdn.link/cdn/PyvesB/eclipse-planet-themes/updatesite

**Comment by PancakeTAS** [comment]
Just tested on a new Eclipse Installation, works perfectly with this link.

**Comment by PyvesB** [comment]
Hopefully fixed for good in 33db153e95b682bc7a54d15a1eaaf51e81cae93a.

**Comment by AObuchow** [comment]
@PyvesB I think gitcdn is currently down? Not sure if this is just some temporary issue, but visiting https://gitcdn.xyz/ seems to give a 503 error, and so does https://gitcdn.link/cdn/PyvesB/eclipse-planet-themes/updatesite/. 

Maybe I'm just making a mistake on my end though.

This seems to be the related gitcdn issue: https://github.com/schme16/gitcdn.xyz/issues/94

Edit: seems to have been a temporary issue :)
